### PR TITLE
[release/6.0] Make tests for FileLoggerProcessor system time independent

### PR DESCRIPTION
**Pull request by wtgodbe**
https://github.com/dotnet/aspnetcore/pull/37755

Seeing if this fixes the test failures we've seen in release/6.0

### CODE REVIEW

**Comment by dougbu** [comment]
Thanks @wtgodbe

Now, 🤞

**Review by dougbu — APPROVED** [review]

**Comment by wtgodbe** [comment]
These 2 tests were added in https://github.com/dotnet/aspnetcore/pull/41492, which was a backport of https://github.com/dotnet/aspnetcore/pull/41375 that had to be fixed manually to account for the fact that https://github.com/dotnet/aspnetcore/pull/37755 wasn't in 6.0. Hopefully backporting that & making those 2 tests the exact same as in `main` will catch whatever made those 2 fail in the first place

**Comment by dougbu** [comment]
Got it, ta

File: src/Middleware/HttpLogging/test/FileLoggerProcessorTests.cs [hunk 1]
```diff
@@ -27,6 +27,8 @@ public class FileLoggerProcessorTests
         private string _messageFour = "Message four";
         private readonly DateTime _today = DateTime.UtcNow;
 
+        private DateTime _today = new DateTime(2021, 01, 01, 12, 00, 00);
```
Comment (dougbu) [inline src/Middleware/HttpLogging/test/FileLoggerProcessorTests.cs:30]:
```suggestion
        private readonly DateTime _today = new DateTime(2021, 01, 01, 12, 00, 00);
```

**Comment by wtgodbe** [comment]
No test failures here, let's see if this fixes other PRs

**Pull request merged**